python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,6 +368,19 @@ def _check_getter(self):
             raise AttributeError('cannot modify or delete "%s" property (declared as "getter")' % _getName(self))
 
     def __get__(self, obj, cls):
+
+        self_name = _getName(self)
+        GangaObject._acquireChildLock(self_name, obj)
```

Comment:
Great this is now giving some nonsense errors about __exit__ from the threading.Lock() object

...time to debug